cfgov/v1/util/migrations.py

```diff
@@ -0,0 +1,59 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import transaction
+from treebeard.mp_tree import MP_Node
+
+
+def get_page(apps, slug):
+    base_page_cls = apps.get_model('wagtailcore', 'Page')
+    return base_page_cls.objects.get(slug=slug)
+
+
+def get_free_path(apps, parent_page):
```

Comment:
This sent me down the rabbit hole of treebeard (which I've never used before) — it's pretty neat, but I agree, I don't see a better way to do this right now.

:+1: for including it in a place we can re-use it. This seems like something we could need to do more often as we start using more data migrations.